scripts/init.sh

```diff
@@ -20,24 +20,46 @@
 
 # Command-line options that can be set in /etc/default/influxdb.  These will override
 # any config file values. Example: "-join http://1.2.3.4:8086"
+DEFAULT=/etc/default/influxdb
+
+# Daemon options
 INFLUXD_OPTS=
 
+# Process name ( For display )
+NAME=influxdb
+
+# User and group
 USER=influxdb
 GROUP=influxdb
 
-if [ -r /lib/lsb/init-functions ]; then
-    source /lib/lsb/init-functions
+# Daemon name, where is the actual executable
+# If the daemon is not there, then exit.
+DAEMON=/opt/influxdb/influxd
+[ -x $DAEMON ] || exit 5
```

Comment:
As far as I know, there isn't standard exit codes in bash except 0,1 and 2 ... I saw you where using exit 5 when the daemon doesn't exist so I use the same when PID is not writable. I add too "exit 4" when config file is not present. I didn't change the exit code when DAEMON doesn't exits.

I respected the exit codes to ensure the backward compatibility and because I think that it doesn't really matter (except for 0 , 1 or 2).